osu.Game/Skinning/BeatmapSkinProvidingContainer.cs

```diff
@@ -24,7 +25,7 @@ protected override bool AllowConfigurationLookup
                 if (beatmapSkins == null)
                     throw new InvalidOperationException($"{nameof(BeatmapSkinProvidingContainer)} needs to be loaded before being consumed.");
 
-                return beatmapSkins.Value;
+                return beatmapColours.Value;
```

Comment:
i don't believe this is correct any more. it is going to block *all* configuration lookups, rather than just colours. this logic would likely need to be moved to a `GetConfig` override which only blocks the combo colour check.

Reply:
That makes sense. I'll revert this and work on an override to handle the combo colour checks instead.